src/FrameReflower/Text.php

```diff
@@ -408,7 +408,7 @@ function get_min_max_width()
                 // split the text into words
                 // The regex splits on everything that's a separator (^\S double negative), excluding nbsp (\xa0), plus dashes
                 // This currently excludes the "narrow nbsp" character
-                $words = array_flip(preg_split('/([^\S\xA0]+|-+)/u', $str, -1, PREG_SPLIT_DELIM_CAPTURE));
+                $words = array_flip(preg_split('/[^\S\xA0]+/u', $str, -1, PREG_SPLIT_DELIM_CAPTURE));
```

Comment:
You've changed how Dompdf splits words within the min/max width calculation logic. But you did not make the same change to the actual line break logic on line [111](https://github.com/dompdf/dompdf/blob/5e6237e26b874f07aed6a81725515496ebbeadc6/src/FrameReflower/Text.php#L111). This will result in inconsistent results when Dompdf is laying out the lines.

Reply:
While this particular problem with the PR is easily fixed I do not believe the change would resolve the reported issue.